### PR TITLE
Update Japanese Translate

### PR DESCRIPTION
**Pull request by neet**
Added Recovery code, Two-factor auth, Video's locales.

### CODE REVIEW

File: app/assets/javascripts/components/locales/ja.jsx [hunk 1]
```diff
@@ -1,121 +1,125 @@
...
-  "status.media_hidden": "非表示のメデイア",
-  "video_player.toggle_sound": "音の切り替え",
-  "account.mention": "@{name} さんに返信",
-  "account.edit_profile": "プロフィールを編集",
-  "account.unblock": "@{name} さんのブロックを解除",
-  "account.unfollow": "フォロー解除",
   "account.block": "@{name} さんをブロック",
-  "account.mute": "ミュート",
-  "account.unmute": "ミュート解除",
+  "account.disclaimer": "このユーザーは他のインスタンスに所属しているため、数字が正確で無い場合があります。",
```
Comment (walf443) [inline app/assets/javascripts/components/locales/ja.jsx]: english is ```This number may be larger.```. So is it correct to "より大きな可能性があります。"?

**Comment by walf443** [comment]
NOTE: This changes conflict to https://github.com/tootsuite/mastodon/pull/1893

**Comment by walf443** [comment]
Can you reset reordering keys?
#1909 #1893 conflict this commit. 

And It's too hard to check changed points for me.

I'd like to change order or translation keys is next step.

**Comment by dabura667** [comment]
@walf443 http://www.mergely.com/LBQaDBhB/
I reordered the pull request so it is easier to read.

PR version is on the right.

**Review by Gargron — APPROVED** [review]

File: app/assets/javascripts/components/locales/ja.jsx (hunk 1 above)
Comment (NeetShin) [inline app/assets/javascripts/components/locales/ja.jsx]: Sure, Thanks

**Pull request merged**